### PR TITLE
feat(create-medusa-app): Add a `--verbose` option.

### PR DESCRIPTION
**Pull request by shahednasser**
## What

Adds a `--verbose` option that shows the output of all underlying processes in real-time.

## Why

This is helpful for testing and debugging issues, especially issues that the community runs into. We can ask community members to pass the `--verbose` option and provide us with the outputted logs if they face problems.

## Caveats

When installing the Next.js starter then terminating the process, the main and child processes don't receive the abort signal as it seems to occur in the child process. This leads to the command continuing but then running into an error in the next step.

As this option is only used for debugging, I don't think it's a big issue.

## Testing

Run the `create-medusa-app` snapshot below with `--verbose` option. Or, change to the `packages/create-medusa-app` directory and run:

```bash
yarn dev --directory-path ~/some-dir --verbose
```

> The `--directory-path` option in this case is necessary as installing the medusa backend in the current `packages/create-medusa-app` directory leads to errors related to yarn workspaces.

### CODE REVIEW

**Review by wangjue666 — APPROVED** [review]

File: packages/create-medusa-app/src/utils/prepare-project.ts [hunk 1]
```diff
@@ -195,12 +200,19 @@ export default async ({
...
-          npxOptions
+        const proc = await execute(
+          [
+            `npx @medusajs/medusa-cli@latest user -e ${admin.email} --invite`,
+            npxOptions,
+          ],
+          { verbose, needOutput: true }
         )
+
+        // console.log(proc)
```
Comment (olivermrbl) [inline packages/create-medusa-app/src/utils/prepare-project.ts]: rm
```suggestion
```

File: packages/create-medusa-app/src/commands/create.ts [hunk 2]
```diff
@@ -225,12 +233,14 @@ export default async ({
...
     if (installNextjs && nextjsDirectory) {
-      void startNextjsStarter({
+      startNextjsStarter({
         directory: nextjsDirectory,
         abortController,
+        verbose,
       })
     }
   } catch (e) {
+    console.log(e)
```
Comment (olivermrbl) [inline packages/create-medusa-app/src/commands/create.ts]: question: rm or use logger?

**Review by olivermrbl — APPROVED** [review]
LGTM

**Pull request merged**